pkgs/applications/blockchains/btcpayserver/update.sh

```diff
@@ -9,4 +9,4 @@ echo "Updating nbxplorer"
 echo
 echo "Updating btcpayserver"
 # Include Razor SDK packages in deps.nix
-../nbxplorer/util/update-common.sh btcpayserver deps.nix '"/p:Configuration="Altcoins-Release" /p:RazorCompileOnBuild=false'
```

Comment:
RazorCompileOnBuild doesn't appear to be needed, but I've only built this on linux-x64

Reply:
Please try again but this time set `altcoinSupport` to `true` in `pkgs/applications/blockchains/btcpayserver/default.nix`.

I am almost sure the build will fail, since you dropped the `Altcoins-Release` flag. The `RazorCompileOnBuild` flag was added in the same commit bf6cac3e7a94de1a5dc0da81899ec489780210a7, so I assume this is also needed when compiling with `altcoinSupport`.

Reply:
`nix-build --expr 'with import <nixpkgs> {}; btcpayserver.override { altcoinSupport = true; }'`

Works for me after restoring using Release, so perhaps it doesn't add any nuget dependencies?

Maybe @nixbitcoin can comment on why this would be needed in restore.

Reply:
Hm, you might be correct. In the meantime, I cherry-picked the commit from this PR into my PR https://github.com/NixOS/nixpkgs/pull/207414

I think we can close this PR and continue discussion in https://github.com/NixOS/nixpkgs/pull/207414

Thank you for your fix!